src/widgets/option-item/ui/option-item.tsx

```diff
@@ -12,7 +13,7 @@ type OptionItemProps = {
 
 export const OptionItem = ({ title, description, linkLabel, href = '' }: OptionItemProps) => (
   <article key={title} className="option">
-    <h3 className="option-title">{title}</h3>
+    <Subtitle className="option-title">{title}</Subtitle>
```

Comment:
Use `subtitleVariants` and remove unnecessary class

Reply:
Done